content/docs/id-tokens.md

```diff
@@ -39,9 +39,10 @@ Because these tokens are signed by Dex and [contain standard-based claims][stand
 * [Kubernetes][kubernetes]
 * [AWS STS][aws-sts]
 
-For details on how to request or validate an ID Token, see [“_Writing apps that use Dex_.”][using-dex]
+For details on how to request or validate an ID Token, see [“_Writing apps that use dex_.”][using-dex]
```

Comment:
Why?

Reply:
To match the link from getting started
https://github.com/dexidp/website/blob/main/content/docs/getting-started.md#further-reading

I thought it would better for them to be equal.

Reply:
We've been pretty haphazard about whether or not to capitalize "Dex." This may be a separate issue: should we capitalize, and if so, how. I seem to recall a discussion around this, and seem to remember that we're trying to standardize on "Dex." I'll try to find the conversation.

Reply:
Thanks, I would appreciate to read the thoughts about this.

As for me, I prefer to use "Dex" writing casual slack posts. But about this link, I checked that "dex" was the first, thus I decided to stick with it.

I will change it to Dex. NP.

Reply:
Yep, lately we are trying to standardize "Dex". It was always inconsistent, "dex" was probably more common, but I prefer "Dex".